### ISSUE TITLE
Please provide example projects

### ISSUE THREAD

**Issue by loesak** [mattfysh/testr.js#17]
I've been trying to get testr working with our testing for a few days now and am having a really difficult time with it. I've ready every blog post, every defect, every forum post i could find, and none of them have eluded to how to get it working properly.

We are using Jasmine (with maven) for our testing. I've created the following GitHub repository and put out a version of a partial example I've seen on a few posts.

Could you please either 1) provide an example project (all files) showing tester working with Jasmine (not mocha) and RequireJS, 2) let me know what i'm doing wrong with the project at the GitHub repository supplied below, or 3) if this project has been superseded by a more mainstream solution (e.g. jasmine-require, squire, isolate, etc) please provide that information.

Thank you.

https://github.com/loesak/jasmine-maven-require-testr

**Comment by mattfysh** [comment]
Hi Aaron,

I'll take a look at this on the weekend, thanks for raising it!

Cheers,
Matt.

**Comment by loesak** [comment]
@mattfysh thank you.

**Comment by hampsterx** [comment]
i'll second that. Have spent more than a day on this and I'm getting to the point of giving up as I am having to poke around testr.js to try and decipher how I should be using it. 

Having issues with multiple deps on a require statement inside testr.run() callback function :(

**Comment by mattfysh** [comment]
@hampsterx the testr.run() method is probably causing too much confusion and I'm in the process of removing it, as well as providing an example of how to setup your test runner. In the meantime, are you able to upload a sample repo and I can have a look at where you're having problems.

**Comment by hampsterx** [comment]
ugh, isolated files to basic core and it works ..

https://github.com/hampsterx/testr_demo

Must be something in my setup, will slowly add stuff back in :P

**Comment by hampsterx** [comment]
If I create a "test3" module and put that as a dependancy onto test2.js it fails _if_ I do a require of more than one module inside testr.run()

Pretty sure that is the source of my problems...

**Comment by mattfysh** [comment]
Yeah that's the issue I found, if you attempt to require() within the testr.run() callback, all hell ensues. Should have v1.4.0 out the door very soon.
